### PR TITLE
Re-enable static member test, and try to make KIP-511 test more reliable

### PR DESCRIPTION
**Pull request by edenhill**

### CODE REVIEW

File: src/rdkafka_sasl_scram.c [hunk 1]
```diff
@@ -934,7 +934,7 @@ static int unittest_scram_nonce (void) {
         rd_kafka_sasl_scram_generate_nonce(&out2);
         RD_UT_ASSERT(out1.size == 32, "Wrong size %d", (int)out2.size);
 
-        RD_UT_ASSERT(memcmp(out1.ptr, out2.ptr, out1.size == 32) != 0,
```
Comment (mhowlett) [inline src/rdkafka_sasl_scram.c]: you should give a more strongly typed language a go. can give a recommendation if you want ;-).

File: tests/CMakeLists.txt [hunk 2]
```diff
@@ -14,6 +14,7 @@ set(
     0012-produce_consume.c
     0013-null-msgs.c
     0014-reconsume-191.c
+    0016-client_swname.c
```
Comment (mhowlett) [inline tests/CMakeLists.txt]: tip: if you use your toes as well as your fingers, you can count up this high.

Comment (mhowlett) [reply tests/CMakeLists.txt]: (buf if you're too lazy too, 16 comes after 15)

File: src/rdkafka_sasl_scram.c (hunk 1 above)
Comment (edenhill) [reply src/rdkafka_sasl_scram.c]: hehe, you said "go".

Newer clang actually warn about this :+1:

File: tests/CMakeLists.txt (hunk 2 above)
Comment (edenhill) [reply tests/CMakeLists.txt]: damnit

**Pull request merged**